src/app/assets/scripts/klorofil-common.js

```diff
@@ -0,0 +1,225 @@
+		function(e) {
+			e.preventDefault();
+
+			// if has scroll
+			if( $(this).parents('.panel').find('.slimScrollDiv').length > 0 ) {
+				affectedElement = $('.slimScrollDiv');
+			}
+
+			$(this).parents('.panel').find(affectedElement).slideUp(300);
+			$(this).find('i.lnr-chevron-up').toggleClass('lnr-chevron-down');
+		},
+		function(e) {
+			e.preventDefault();
+
+			// if has scroll
+			if( $(this).parents('.panel').find('.slimScrollDiv').length > 0 ) {
+				affectedElement = $('.slimScrollDiv');
+			}
+
+			$(this).parents('.panel').find(affectedElement).slideDown(300);
+			$(this).find('i.lnr-chevron-up').toggleClass('lnr-chevron-down');
+		}
+	);
+
+
+	/*-----------------------------------/
+	/*	PANEL SCROLLING
+	/*----------------------------------*/
+
+	if( $('.panel-scrolling').length > 0) {
+		$('.panel-scrolling .panel-body').slimScroll({
+			height: '430px',
+			wheelStep: 2,
+		});
+	}
+
+	if( $('#panel-scrolling-demo').length > 0) {
+		$('#panel-scrolling-demo .panel-body').slimScroll({
+			height: '175px',
+			wheelStep: 2,
+		});
+	}
+
+	/*-----------------------------------/
+	/*	TODO LIST
+	/*----------------------------------*/
+
+	$('.todo-list input').change( function() {
+		if( $(this).prop('checked') ) {
+			$(this).parents('li').addClass('completed');
+		}else {
+			$(this).parents('li').removeClass('completed');
+		}
+	});
+
+
+	/*-----------------------------------/
+	/* TOASTR NOTIFICATION
+	/*----------------------------------*/
+
+	if($('#toastr-demo').length > 0) {
+		toastr.options.timeOut = "false";
+		toastr.options.closeButton = true;
+		toastr['info']('Hi there, this is notification demo with HTML support. So, you can add HTML elements like <a href="#">this link</a>');
+
+		$('.btn-toastr').on('click', function() {
+			$context = $(this).data('context');
+			$message = $(this).data('message');
+			$position = $(this).data('position');
+
+			if($context == '') {
+				$context = 'info';
+			}
+
+			if($position == '') {
+				$positionClass = 'toast-left-top';
+			} else {
+				$positionClass = 'toast-' + $position;
+			}
+
+			toastr.remove();
+			toastr[$context]($message, '' , { positionClass: $positionClass });
+		});
+
+		$('#toastr-callback1').on('click', function() {
+			$message = $(this).data('message');
+
+			toastr.options = {
+				"timeOut": "300",
+				"onShown": function() { alert('onShown callback'); },
+				"onHidden": function() { alert('onHidden callback'); }
+			}
+
+			toastr['info']($message);
+		});
+
+		$('#toastr-callback2').on('click', function() {
+			$message = $(this).data('message');
+
+			toastr.options = {
+				"timeOut": "10000",
+				"onclick": function() { alert('onclick callback'); },
+			}
+
+			toastr['info']($message);
+
+		});
+
+		$('#toastr-callback3').on('click', function() {
+			$message = $(this).data('message');
+
+			toastr.options = {
+				"timeOut": "10000",
+				"closeButton": true,
+				"onCloseClick": function() { alert('onCloseClick callback'); }
```

Comment:
why using Jquery when you are using react ?
do we really need 2 lib ??
whats the need of jquery here??

Reply:
ok sir will remove it